app/views/unattended/provisioning_templates/snippet/saltstack_minion.erb

```diff
@@ -11,6 +11,9 @@ description: |
 master: <%= host_param('salt_master') %>
 log_level: warning
 
+autosign_grains:
+  - autosign_key
```

Comment:
you still need to update snapshots (`test/unit/foreman/renderer/snapshots/ProvisioningTemplate/snippet/saltstack_minion.host4dhcp.snap.txt` to be precise) for this change!